src/muiThemeable.js

```diff
@@ -1,23 +1,39 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
+import merge from 'lodash.merge';
+
+const defaultTheme = getMuiTheme();
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
-export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+export default function muiThemeable(WrappedComponent, forwardMethods) {
+
+  const methods = {};
+  if (forwardMethods) {
+    forwardMethods.map(name => methods[name] =
+      (function(...args) { return this.refs.WrappedComponent[name](...args); }));
+  }
+
+  return React.createClass(merge({
```

Comment:
we have this dependency else where. But alright. I'll change this one, makes more sense.

Thanks for the feed backs :+1: